client/src/util/helpers.ts

```diff
@@ -188,4 +227,12 @@ export default {
     });
     return arr.join(" ");
   },
+
+  getTranslationOfComplaintType(complaintType: string, i18n: I18n) {
+    const translatedType = hpdComplaintTypeTranslations.get(complaintType);
```

Comment:
I'm curious - instead of having the map above, could you just do

```
const translatedType = t`{complaintType}`
```
or something like that? Would that work?

Reply:
Ah unfortunately we can't do <code>t`{complaintType}`</code> because (IIRC) `t` is actually a [babel macro](https://babeljs.io/blog/2017/09/11/zero-config-with-babel-macros) provided by lingui, which basically means that it doesn't behave like a normal piece of code--as @sraby mentioned, it's used when we run `yarn lingui:extract` to extract literal strings to be translated, and it basically needs to have every single potential string spelled out for it.